### ISSUE TITLE
Autonomous database's network ACL updated when no changes made

### ISSUE THREAD

**Issue by anggol** [terraform-providers/terraform-provider-oci#1050]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Terraform Version and Provider Version

Terraform v0.12.24
provider.oci v3.73.0

### Affected Resource(s)

oci_database_autonomous_database

### Terraform Configuration Files

```
resource "oci_database_autonomous_database" "autonomous_dbtest" {
  #Required
  admin_password           = var.autonomous_database_admin_password
  compartment_id           = data.terraform_remote_state.common.outputs.onetooldev_compartment
  cpu_core_count           = var.autonomous_database_cpu_core_count
  data_storage_size_in_tbs = var.autonomous_database_data_storage_size_in_tbs
  db_name                  = "onetooldbtest"

  #Optional
  db_version                                     = var.autonomous_database_version
  db_workload                                    = var.autonomous_database_db_workload
  display_name                                   = "${var.prefix_name}-autonomous-dbtest"
  is_auto_scaling_enabled                        = "true"
  license_model                                  = var.autonomous_database_license_model
  is_preview_version_with_service_terms_accepted = "false"
  whitelisted_ips                               = ["160.34.126.0/23", "160.34.115.0/24", "160.34.108.0/23"]
}
```

### Expected Behavior

First terraform apply creates the database. Subsequent terraform applys, with no changes made, should detect that no changes are required and shouldn't try to apply any updates.

### Actual Behavior

First terraform apply created the database. Subsequent terraform applys identify that changes are required to the Network Access Control List (whitelisted_ips), try to apply them and an error is returned because the list has already been applied.

### Steps to Reproduce

1. `terraform apply`

```
Terraform will perform the following actions:

  # oci_database_autonomous_database.autonomous_dbtest will be created
  + resource "oci_database_autonomous_database" "autonomous_dbtest" {
...
      + whitelisted_ips                                = [
          + "160.34.126.0/23",
          + "160.34.115.0/24",
          + "160.34.108.0/23",
        ]
    }

Plan: 1 to add, 0 to change, 0 to destroy.

Do you want to perform these actions?
  Terraform will perform the actions described above.
  Only 'yes' will be accepted to approve.

  Enter a value: yes

oci_database_autonomous_database.autonomous_dbtest: Creating...
...
oci_database_autonomous_database.autonomous_dbtest: Creation complete after 1m29s [id=ocid1.autonomousdatabase.oc1.iad.abuwcljtzd7oa2op2aden2ta5zcy4alidqksl3ytx3iygufi6xy6tcwbsm5a]

Apply complete! Resources: 1 added, 0 changed, 0 destroyed.
```

2. `terraform apply`

```
An execution plan has been generated and is shown below.
Resource actions are indicated with the following symbols:
  ~ update in-place

Terraform will perform the following actions:

  # oci_database_autonomous_database.autonomous_dbtest will be updated in-place
  ~ resource "oci_database_autonomous_database" "autonomous_dbtest" {
...
      ~ whitelisted_ips                                = [
          + "160.34.126.0/23",
            "160.34.115.0/24",
            "160.34.108.0/23",
          - "160.34.126.0/23",
        ]
    }

Plan: 0 to add, 1 to change, 0 to destroy.

Do you want to perform these actions?
  Terraform will perform the actions described above.
  Only 'yes' will be accepted to approve.

  Enter a value: yes

oci_database_autonomous_database.autonomous_dbtest: Modifying... [id=ocid1.autonomousdatabase.oc1.iad.abuwcljtzd7oa2op2aden2ta5zcy4alidqksl3ytx3iygufi6xy6tcwbsm5a]

Error: Service error:InvalidParameter. This network ACL has already been applied to the Autonomous Database.. http status code: 400.
```

### Important Factoids

The tfstate file includes the following:

"whitelisted_ips": [
  "160.34.115.0/24",
  "160.34.108.0/23",
  "160.34.126.0/23"
]

**Comment by abhilash-av** [comment]
@anggol thanks for reporting it, The fix is expected to be available in Provider Version 3.75.0.

**Comment by anggol** [comment]
Thank you abhilash-av.

You mentioned the fix is expected to be available in Provider Version 3.75.0. If possible, can you give me a rough idea of when this is scheduled to be released?

**Comment by abhilash-av** [comment]
@anggol 

Provider Version 3.75.0 was released today. 
Closing this issue, let us know if you have any other questions.

Thanks
